react/provider/ConfigProvider.tsx

```diff
@@ -1,15 +1,18 @@
-import React, { useState, useEffect } from "react";
-import { useMutation, useQuery } from "react-apollo";
+import React, { useState, useEffect } from 'react'
+import { useMutation, useQuery } from 'react-apollo'
 
-import settingsSchema from "../queries/settingsSchema.gql";
-import saveMappSettings from "../queries/saveSettings.gql";
+import settingsSchema from '../queries/settingsSchema.gql'
+import saveMappSettings from '../queries/saveSettings.gql'
 
 const defaultConfigValues = {
-  tiId: "0",
-  tiResponder: "0",
-  acId: "0",
-  acM: "0",
-};
+  tiId: '0',
+  tiResponder: '0',
+  acId: '0',
+  acM: '0',
+  engageApiUrl: '',
```

Comment:
We must have a value here, an empty string will cause the bug I mentioned in our call